### PR TITLE
[tests] Add tests for Console APIs

### PR DESCRIPTION
**Pull request by cuiyanx**
TOTAL: 1 of 1 passed

Signed-off-by: Cui, Yan yanx.cui@intel.com

### CODE REVIEW

**Comment by jimmy-huang** [comment]
+1

File: tests/test-console-time-manual.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+for (var i = 1; i < 21; i++) {
+    console.time("Timer" + i);
+}
+
+setTimeout(function() {
+    for (var j = 1; j < 20; j++) {
+        console.timeEnd("Timer" + j);
+
+        var after = performance.now();
+        var diff = after - before;
```
Comment (grgustaf) [inline tests/test-console-time-manual.js]: Add a "| 0" to the end of this line to round it off to the nearest ms which will make the output more readable.

File: tests/test-console-time-manual.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    var Timer21;
+    try {
+        console.time(Timer21);
+        console.timeEnd(Timer21);
+    } catch (e) {
+        console.log("expected result: throw error");
+        console.log("ERROR: " + e.name + " - " + e.message + "\n");
+    }
+
+    console.log("test: set timer name as null");
```
Comment (cuiyanx) [inline tests/test-console-time-manual.js]: UR right.

**Comment by grgustaf** [comment]
+1

**Pull request merged**